tests/benchdnn/graph/ref_partition.cpp

```diff
@@ -70,8 +70,14 @@ ref_partition_t::ref_partition_t(const deserialized_graph_t &dg,
     }
 };
 
-int ref_partition_t::init_ref(const std::vector<size_t> &graph_in_ports,
-        partition_mem_map_t &partition_mem_map, res_t *res) {
+int ref_partition_t::init_ref(
+        const std::vector<size_t> &graph_in_ports, res_t *res) {
+
+    // Not create reference primitives and filling data with pre-designed
+    // strategies for `no_ref_memory`
+    if (!has_bench_mode_bit(mode_bit_t::corr)
+            && has_bench_mode_modifier(mode_modifier_t::no_ref_memory))
+        return OK;
```

Comment:
This should be sufficient...
```suggestion
    if (!has_bench_mode_bit(mode_bit_t::corr))
        return OK;
```